core/embed/projects/prodtest/README.md

```
@@ -1466,6 +1466,25 @@ OK
 
 Runs a Tropic stress test that repeatedly calls `lt_init()`, `lt_session_start()`, `lt_mac_and_destroy()`, `lt_ecc_key_generate()` and `lt_random_value_get()` to test that Tropic doesn't enter alarm mode.
 
+### tropic-benchmark
+
+Measures the actual average duration of individual Tropic operations over 25 iterations and prints the results.
+
+Example:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)